INS0901-INS1000/INS0970LeipzigUB.xml

```diff
@@ -0,0 +1,56 @@
+<?xml version="1.0" encoding="UTF-8"?><?xml-model href="https://raw.githubusercontent.com/BetaMasaheft/schema/master/tei-betamesaheft.rng"
+schematypens="http://relaxng.org/ns/structure/1.0"?><?xml-model href="https://raw.githubusercontent.com/BetaMasaheft/schema/master/tei-betamesaheft.rng"
+type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?><TEI xmlns="http://www.tei-c.org/ns/1.0" xml:id="INS0970LeipzigUB" xml:lang="en" type="ins">
+  <teiHeader xml:base="https://betamasaheft.eu/">
+      <fileDesc>
+         <titleStmt>
+            <title>Universitätsbibliothek Leipzig</title>
+            <editor key="VP"/>
+            <editor role="generalEditor" key="AB"/>
+            <funder></funder>
+         </titleStmt>
+         <publicationStmt>
+            <authority>Hiob-Ludolf-Zentrum für Äthiopistik</authority>
+            <publisher>Die Schriftkultur des christlichen Äthiopiens und Eritreas: Eine multimediale Forschungsumgebung / Beta maṣāḥǝft</publisher>
+            <pubPlace>Hamburg</pubPlace>
+
+            <availability>
+               <licence target="http://creativecommons.org/licenses/by-sa/4.0/">
+                  <p>
+            This file is licensed under the Creative Commons
+            Attribution-ShareAlike 4.0. </p>
+               </licence>
+            </availability>
+
+         </publicationStmt>
+         <sourceDesc>
+            <p>A TEI file created from original data</p>
+         </sourceDesc>
+      </fileDesc>
+      <encodingDesc>
+         <xi:include xmlns:xi="http://www.w3.org/2001/XInclude" href="https://raw.githubusercontent.com/BetaMasaheft/Documentation/master/prefixDef.xml">
+            <xi:fallback>
+               <p>Definitions of prefixes used.</p>
+            </xi:fallback>
+         </xi:include>
+      </encodingDesc>
+      <profileDesc>
+         <langUsage><language ident="en">English</language><language ident="de">German</language></langUsage>
+      </profileDesc>
+      <revisionDesc>
+        <change who="VP" when="2022-03-22">Created entity</change>
+
+       </revisionDesc>
+  </teiHeader>
+  <text xml:base="https://betamasaheft.eu/">
+      <body>
+         <listPlace>
+            <place  type="library" subtype="institution">
```

Comment:
looks very good, it would be great if you could also add to the `place`
the attribute `sameAs="wd:Q872896"` to have a link to wikidata, that is
`<place  type="library" subtype="institution" sameAs="wd:Q872896">`